packages/apps/ability-sol-transaction-signer/src/lib/lit-action-helpers/signSolanaTransaction.ts

```diff
@@ -0,0 +1,36 @@
+import { Transaction, VersionedTransaction, Keypair } from '@solana/web3.js';
+import { ethers } from 'ethers';
+
+export function signSolanaTransaction({
+  solanaKeypair,
+  transaction,
+}: {
+  solanaKeypair: Keypair;
+  transaction: Transaction | VersionedTransaction;
+}): string {
+  try {
```

Comment:
🤔 maybe we should have used this pattern in the main wrapped keys SDK instead of that optional argument 'isVersioned...' :(